client/app/bundles/course/assessment/submission/localStorage/liveFeedbackChat/operations.ts

```diff
@@ -0,0 +1,49 @@
+import { LiveFeedbackChatData } from '../../types';
+
+const getLocalStorageKey = (
+  questionId: string | number,
```

Comment:
previously we commonly do this since the return type of `questionId` from the backend is sometimes integer and sometimes string; but now we use `answerId`, we can make sure that all types returning are number